src/components/IpfsInput/IpfsInput.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { useMoralisFile } from 'react-moralis';
+import { IpfsInputprops } from './types';
+import { Upload } from '../Upload';
+
+const IpfsInput: React.FC<IpfsInputprops> = ({
+    Theme = 'textOnly',
+
+    ...props
```

Comment:
could you destructure props in type file and give them default values if they are optional 😅